I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 156, in script
    with noisedissolve
  File "game/script.rpy", line 156, in <module>
    with noisedissolve
NameError: name 'noisedissolve' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 156, in script
    with noisedissolve
  File "C:\Users\Martin\Documents\Martin Gabriel\Winnie\Renpy\renpy-6.99.5-sdk\renpy\ast.py", line 1243, in execute
    trans = renpy.python.py_eval(self.expr)
  File "C:\Users\Martin\Documents\Martin Gabriel\Winnie\Renpy\renpy-6.99.5-sdk\renpy\python.py", line 1482, in py_eval
    return eval(py_compile(source, 'eval'), globals, locals)
  File "game/script.rpy", line 156, in <module>
    with noisedissolve
NameError: name 'noisedissolve' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.5.602
Mi color 0.0
